.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: CI Build
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    name: Check Build
+    runs-on: macOS-10.14
+    steps:
+      - uses: actions/checkout@master
+      - name: Install dependencies
+        working-directory: ./actions
+        run: npm install
+      - name: Check version
+        id: check_version
+        uses: ./actions/check_version
+      - name: Build 
+        run: |
+         carthage update
+         xcodebuild -project kintone-ios-sdk.xcodeproj -scheme kintone-ios-sdk -destination 'platform=iOS Simulator,name=iPhone Xs Max' CONFIGURATION_BUILD_DIR=release/
```

Comment:
@Pham-Gia-Huong 
Please, research more about build for release and archiving.
These lines should be changed into something like following
xcodebuild -scheme <ReleaseScheme> archive
After this command, there should be an archive ready for upload.
You should test if you can use the archive.